src/useConditionalEffect/__docs__/example.stories.tsx

```diff
@@ -1,54 +1,37 @@
 import * as React from 'react';
 import { useState } from 'react';
-import { useConditionalEffect, useToggle, useUpdateEffect } from '../..';
+import { useConditionalEffect, useUpdateEffect } from '../..';
 
 export const Example: React.FC = () => {
-  const [isToggled, toggle] = useToggle(false);
+  const [state1, setState1] = useState(2);
+  const [state2, setState2] = useState(2);
 
-  // eslint-disable-next-line react/no-unstable-nested-components
-  const ToggledComponent: React.FC = () => {
-    const [state1, setState1] = useState(2);
-    const [state2, setState2] = useState(2);
-
-    useConditionalEffect(
-      () => {
-        // eslint-disable-next-line no-alert
-        alert('COUNTERS VALUES ARE EVEN');
-      },
-      [state1, state2],
-      [state1, state2],
-      (conditions) => conditions.every((i) => i && i % 2 === 0),
-      useUpdateEffect
-    );
-
-    return (
-      <div>
-        <div>Alert will be displayed when both counters values are even</div>
-        <div>Effect also invoked on initial mount</div>
-        <button
-          onClick={() => {
-            setState1((i) => i + 1);
-          }}>
-          increment counter 1 [{state1}]
-        </button>{' '}
-        <button
-          onClick={() => {
-            setState2((i) => i + 1);
-          }}>
-          increment counter 2 [{state2}]
-        </button>
-      </div>
-    );
-  };
-
-  if (isToggled) {
-    return <ToggledComponent />;
-  }
+  useConditionalEffect(
+    () => {
+      // eslint-disable-next-line no-alert
+      alert('VALUES OF THE COUNTERS ARE EVEN');
+    },
+    [state1, state2],
+    [state1, state2],
+    (conditions) => conditions.every((i) => i && i % 2 === 0),
+    useUpdateEffect
+  );
 
   return (
     <div>
-      <div>As example component displays alert right from mount - it is initially unmounted.</div>
-      <button onClick={() => toggle()}>Mount example component</button>
+      <div>An alert will be displayed when both counters have even values.</div>
+      <button
+        onClick={() => {
+          setState1((i) => i + 1);
+        }}>
+        Increment counter 1 [{state1}]
+      </button>{' '}
+      <button
+        onClick={() => {
+          setState2((i) => i + 1);
+        }}>
+        Increment counter 2 [{state2}]
+      </button>
     </div>
   );
 };
```

Comment:
When reading throught this example and trying to improve the language, I realised that toggled example component here was really confusing and it did not even work as intended, since no alert was shown when the example component was mounted. Thus, I decided to remove it.